### DISCUSSION TITLE
Network graph was broken with github pages

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [community/community#52481] [Actions] [answered]
### Select Topic Area

Question

### Body

I'm not sure what happened, but when I woke up, my special repository had only one commit again. I'm really frustrated about it. You can see the repository at https://github.com/Anmol-Baranwal/Anmol-Baranwal/tree/main.

I'm trying to figure out what caused this issue. Could it be related to dependabot.yml? I need to know so that I can avoid similar problems in the future at all costs.

My Network graph was broken: See [here](https://github.com/Anmol-Baranwal/Anmol-Baranwal/network)
![image](https://user-images.githubusercontent.com/74038190/231055005-a095cda9-77a9-459e-aa5d-79ab4695c3f9.png)

You can view the earlier commits that were made here: https://github.com/Anmol-Baranwal/Anmol-Baranwal/tree/hack.

<br>

You can see here that the repository was old enough. When I made a pull request, it gave an error message saying that the branches have completely different commit histories. Please refer to the attached image: ![image](https://user-images.githubusercontent.com/74038190/231051716-0ea19d08-a898-4bae-af93-d0eba5ce0f55.png)

Please find attached a screenshot of my profile: 
![github com_Anmol-Baranwal](https://user-images.githubusercontent.com/74038190/231049363-6e4315d5-7de7-4411-9b69-155eb6b1e1f8.png)

**Comment by airtower-luna** [comment, marked as answer]
The orphan commits have one thing in common: They look like they were created by an Actions workflow. Based on your recent runs and the commit ID it seems to be this one (mind that the logged commit ID is the same as the current HEAD of `main`): https://github.com/Anmol-Baranwal/Anmol-Baranwal/actions/runs/4667481543/jobs/8263331865

So you'll have to disable that or fix the workflow (or rather the action it calls).

**Reply by Anmol-Baranwal** [reply]
I fixed the problem and returned to answer, only to discover that you had already provided a response. Nonetheless, I managed to identify my mistake after having my profile reset twice :)

Even though I will mark your answer, could you please provide additional context as to why the orphan commits have caused issues with the repository's history and reset everything to an earlier state? Also, could you explain how you identified the error?

```yml
target_branch: main
```
This caused the error.

**Reply by airtower-luna** [reply]
From the history (`main` branch with exactly one commit) it was clear that *something* must've created an empty (not connected to current history, thus "orphan") branch, made that one commit, and force-pushed that branch as `main`. That leaves the question: *What* did that? The commit author pointed at one of your many Actions workflows, so I poked around in the recent logs to find one that lined up with the commit time and the message. :wink:

**Reply by Anmol-Baranwal** [reply]
Thanks for the explanation.
